cfg/plus4-hires.cfg

```diff
@@ -0,0 +1,57 @@
+# Linker configuration that allows for a hi-res bitmap at $C000-$DF3F, but
+# puts the stack (and a "HIBSS" segment) in the remaining RAM at $DF40-$FD00.
+
+FEATURES {
+    STARTADDRESS: default = $1001;
+}
+SYMBOLS {
+    __LOADADDR__:  type = import;
+    __EXEHDR__:    type = import;
+    __STACKSIZE__: type = weak, value = $0800; # 2k stack
+    __HIMEM__:     type = weak, value = $FD00;
+    __RESERVED_C000__: type = weak, value = 8000; # Reserve 8000 bytes for hi-res bitmap
+}
+MEMORY {
+    # Memory reserved for bitmap
+    RESERVED: file = "", define = yes, start = $C000,           size = __RESERVED_C000__;
+
+    ZP:       file = "", define = yes, start = $0002,           size = $001A;
+    LOADADDR: file = %O,               start = %S - 2,          size = $0002;
+    HEADER:   file = %O, define = yes, start = %S,              size = $000D;
+    MAIN:     file = %O, define = yes, start = __HEADER_LAST__, size = __RESERVED_START__ - __MAIN_START__;
+
+    # Space between bitmap and top of memory
+    HIRAM:    file = "", define = yes, start = __RESERVED_LAST__, size = __HIMEM__ - __HIRAM_START__ - __STACKSIZE__;
+}
+SEGMENTS {
+    ZEROPAGE: load = ZP,       type = zp;
+    LOADADDR: load = LOADADDR, type = ro;
+    EXEHDR:   load = HEADER,   type = ro;
+    STARTUP:  load = MAIN,     type = ro;
+    LOWCODE:  load = MAIN,     type = ro,  optional = yes;
+    CODE:     load = MAIN,     type = ro;
+    ONCE:     load = MAIN,     type = ro,  optional = yes;
+    RODATA:   load = MAIN,     type = ro;
+    DATA:     load = MAIN,     type = rw;
+    INIT:     load = MAIN,     type = bss;
+    BSS:      load = MAIN,     type = bss, define   = yes;
+
+    # Allow data between bitmap and top of memory to be used as a second BSS
+    # space. Define symbols for it so that it can be supplied to _heapadd().
+    HIBSS:    load = HIRAM,    type = bss, optional = yes, define   = yes;
```

Comment:
The additional blanks between `define` and `= yes` aren't valid here as there's nothing to align to.